ui/app/src/Views/AlertList/AlertView.elm

```diff
@@ -0,0 +1,91 @@
+module Views.AlertList.AlertView exposing (view)
+
+import Alerts.Types exposing (Alert)
+import Html exposing (..)
+import Html.Attributes exposing (class, style, href)
+import Html.Events exposing (onClick)
+import Types exposing (Msg(CreateSilenceFromAlert, Noop, MsgForAlertList))
+import Utils.Date
+import Views.AlertList.Types exposing (AlertListMsg(AddFilterMatcher))
+import Utils.Views exposing (buttonLink)
+import Utils.Filter
+import Time exposing (Time)
+
+
+view : Alert -> Html Msg
+view alert =
+    li
+        [ class "align-items-center list-group-item alert-list-item p-0 d-inline-flex justify-content-start"
+        ]
+        [ dateView alert.startsAt
+        , labelButtons alert.labels
+        , div [ class "ml-auto d-inline-flex align-self-stretch p-2", style [ ( "border-left", "1px solid #ccc" ) ] ]
+            [ generatorUrlButton alert.generatorUrl
+            , silenceButton alert
+            ]
+        ]
+
+
+dateView : Time -> Html Msg
+dateView time =
+    i
+        [ class "h-100  d-flex flex-column justify-content-center p-2 text-muted"
+        , style [ ( "border-right", "1px solid #ccc" ), ( "font-family", "monospace" ) ]
+        ]
+        [ span [] [ text <| Utils.Date.timeFormat time ]
+        , small [] [ text <| Utils.Date.dateFormat time ]
+        ]
+
+
+labelButtons : List ( String, String ) -> Html Msg
+labelButtons labels =
+    let
+        -- the alertname label should be first
+        sortedLabels =
+            List.append
```

Comment:
Wow, this is beautiful. But @fabxc suggested to not render the label key if it is "alertname" to shorten the label list presentation. So the alertname label needs to have a different `labelButton` render function then any other label (See current implementation). I will use the partition function though.

But I will definitely